### PR TITLE
[Translation] State of html initial setup

### PR DESCRIPTION
**Pull request by romulo1984**
Added the necessary files for the translation of the State of HTML 2023, which will allow you to receive the translation from other collaborators.

### CODE REVIEW

**Review by rodrigoant — APPROVED** [review]
🚀

File: state_of_html.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      Novos elementos como `<selectlist>` estão no horizonte, novos recursos interessantes como popovers, e uma série de APIs do navegador relacionadas (Web Components, PWAs, etc.).
+
+      Este ano, [Lea Verou](https://lea.verou.me/) assumiu a formidável tarefa de liderar o design desta nova pesquisa, desde o conteúdo à UX e muito mais.
+
+      Com base no sucesso do [Estado do JS](http://stateofjs.com/) e do [Estado do CSS](https://stateofcss.com/), apresentamos o **Estado do HTML**;
+      a última peça que faltava para completar a trilogia, para que possamos acompanhar a evolução da plataforma web como um todo.
+
+      Benefícios para você:
+      - Os resultados da pesquisa são **usados por navegadores** e grupos de padrões **para priorização de roteiros**.
+      Suas respostas podem ajudar a implementar recursos de seu interesse, priorizar incompatibilidades de navegador e resolver lacunas na plataforma.
```
Comment (vilaboim) [inline state_of_html.yml:23]: Hi, @romulo1984 
First, congrats for the translation.
I just joined the group and would like to contribute to your PR.

The only thing I believe we can improve here is the use of the word "recursos", I think a more accurate translation would be "funcionalidades", and this way we don't repeat the word in different contexts (I'll give an example in a following comment).

So this sentence would be: "Suas respostas podem ajudar a implementar funcionalidades de seu interesse, priorizar incompatibilidades de navegador e resolver lacunas na plataforma."

File: state_of_html.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      - Aprenda sobre recursos novos e futuros; adicione recursos à sua lista de leitura e obtenha uma lista de recursos no final!
```
Comment (vilaboim) [inline state_of_html.yml:24]: And this sentence would be: "Aprenda sobre funcionalidades novas e futuras; adicione funcionalidades à sua lista de leitura e obtenha uma lista de recursos no final!"

File: state_of_html.yml (hunk 1 above)
Comment (romulo1984) [reply state_of_html.yml]: Thanks for the suggestions, @vilaboim. I pushed the recommended changes.

**Pull request merged**